Pinta/MainWindow.cs

```diff
@@ -129,14 +129,17 @@ private void DockNotebook_TabClosed (object? sender, TabClosedEventArgs e)
 	{
 		var view = (DocumentViewContent) e.Item;
 
-		if (PintaCore.Workspace.OpenDocuments.IndexOf (view.Document) > -1) {
-			PintaCore.Workspace.SetActiveDocument (view.Document);
-			PintaCore.Actions.File.Close.Activate ();
+		if (PintaCore.Workspace.OpenDocuments.IndexOf (view.Document) <= -1)
+			return;
 
-			// User must have canceled the close
-			if (PintaCore.Workspace.OpenDocuments.IndexOf (view.Document) > -1)
-				e.Cancel = true;
-		}
+		PintaCore.Workspace.SetActiveDocument (view.Document);
+		PintaCore.Actions.File.Close.Activate ();
+
+		// User must have canceled the close
```

Comment:
The comment should now go after the if statement since it goes along with setting `Cancel` to true